SmallCommitsWorkshop/Controllers/UsersController.cs

```diff
@@ -27,5 +27,41 @@ public class UsersController : Controller {
 			IEnumerable<User> users = await m_usersContext.Users.ToListAsync();
 			return users.ToDictionary( user => user.Id, user => user );
 		}
+
+		[HttpGet( "{userId}" )]
+		public async Task<ActionResult<IDictionary<long, User>>> Get( long userId ) {
+			User user = await m_usersContext.Users.FindAsync( userId );
+			if( user == null ) {
+				return NotFound();
+			}
+			return new Dictionary<long, User>( 1 ) { { user.Id, user } };
+		}
+
+		[HttpPost]
+		public async Task<ActionResult> CreateOrUpdate( [FromBody] User newUser ) {

```

Comment:
I'm being pedantic here, but I can silence this thought: the caller probably shouldn't get to choose the user id.  If they do get to choose the user id, this could be `HttpPut`.

Alright, now that I've got that out, let's ignore it.

Reply:
I definitely got lazy here and did not want to create another PONO to represent the incoming payload.